scripts/palette.js

```diff
@@ -0,0 +1,12 @@
+const flatten = (obj, prefixes = []) => Object.entries(obj)
+  .reduce((prev, [key, val]) => ((typeof val === 'string') ?
+    {
+      ...prev,
+      [[...prefixes, key].join('-').replace('neutral-', 'brightness-')]: val,
```

Comment:
```suggestion
      [[...prefixes, key].join('-')]: val,
```

This is in place now to avoid renaming all the neutrals in a huge unmergeable swoop. Will update before merging